parsers/perl6.c

```diff
@@ -325,7 +325,8 @@ findPerl6Tags (void)
 parserDefinition *
 Perl6Parser (void)
 {
-    static const char *const extensions[] = { "p6", "pm6", "pm", "pl6", NULL };
+    static const char *const extensions[] = { "p6", "pm6", "pm", "pl6",
+		"pod6", "t6", "raku", "rakumod", "rakudoc", "rakutest", NULL };
```

Comment:
In  my understanding pod6 and rakumod are for extensions for a documentation language, not for Perl6.
I guess our perl6 parser doesn't handle input fils for the documentation language.